internal/config/mysql.go

```diff
@@ -60,3 +66,39 @@ func (m *MySQL) Bind() *MySQL {
 	m.InitialPingDuration = GetActualValue(m.InitialPingDuration)
 	return m
 }
+
+func (m *MySQL) Opts() ([]mysql.Option, error) {
```

Comment:
I am not sure but should we put this logic to the `New()` function instead of `Opts()`?

Reply:
Which package do you think should have it? I don't know a better package than this one. :thinking:
It is just a conversion from a `struct` to a `[]Option` type, so i think it's no problem to be here.